tests/queries/0_stateless/01018_ddl_dictionaries_select.sql

```diff
@@ -139,6 +139,6 @@ SELECT {CLICKHOUSE_DATABASE:String} || '.dict3' as n, dictGet(n, 'some_column',
 
 DROP TABLE {CLICKHOUSE_DATABASE:Identifier}.table_for_dict;
 
-SYSTEM RELOAD DICTIONARIES; -- {serverError UNKNOWN_TABLE}
+SYSTEM RELOAD DICTIONARY {CLICKHOUSE_DATABASE:Identifier}.dict3; -- {serverError UNKNOWN_TABLE}
```

Comment:
What about other tests which use `SYSTEM RELOAD DICTIONARIES` ? I see `01018_dictionaries_from_dictionaries.sql` and `02179_dict_reload_on_cluster.sql`.

Reply:
I checked all of them.